kong/runloop/handler.lua

```diff
@@ -707,6 +709,20 @@ return {
             header[upstream_status_header] = matches[0]
           end
         end
+
+        if cookie_hash_data then
+          local cookie, ok, err
```

Comment:
There is no benefits here in separating the declaration and initialization of those variables. Let's keep things more concise and declare/initialize them on the same line (by simply moving the `local` keyword to the below lines).